src/pages/DataAccessRequestApplication_new.js

```diff
@@ -959,18 +959,8 @@ class DataAccessRequestApplicationNew extends Component {
             div({className: 'step-container'}, [
               h(DataUseAgreements, {
                 darCode: darCode,
-                problemSavingRequest,
-                attestAndSave: this.attestAndSave,
-                ConfirmationDialogComponent,
-                partialSave: this.partialSave,
-                prevPage: this.prevPage,
-                step1Invalid,
-                step2Invalid,
-                step3Invalid,
-                libraryCardInvalid,
-                showValidationMessages,
```

Comment:
Are we removing the invalid checks and problemSavingRequest and ConfirmationDialogComponent completely? When running locally my console shows a few warning about these being assigned but never used.

Reply:
Good point, I can remove those. This is part one of the PR, so I didn't want to do too much, but I don't want to add linting issues to develop, so I will get rid of those.